kernelci/api/models.py

```diff
@@ -203,6 +203,9 @@ class Node(DatabaseModel):
     jobfilter: Optional[List[str]] = Field(
         description="Restrict jobs that can be scheduled by this node"
     )
+    platformfilter: Optional[List[str]] = Field(
```

Comment:
I'd suggest using `_` between words for better readability as per python naming conventions.
`platform_filter`.

Reply:
Done. I will prefer to keep in json node consistent with previous styling, without underscore, and i will rename later also in code treeid and jobfilter as well in python code (keeping in json same, as migration is not working at moment).